rosdep/python.yaml

```diff
@@ -865,7 +865,9 @@ python-bottle:
   debian: [python-bottle]
   fedora: [python-bottle]
   gentoo: [dev-python/bottle]
-  ubuntu: [python-bottle]
+  ubuntu: 
+    '*': [python-bottle]
+    'focal': [python3-bottle]
```

Comment:
We shouldn't make this change here, as the `python-` keys are implicitly for Python 2.

Instead, what we should do is to add in a new key which is `python3-bottle`, and then add in the python3 package entries as appropriate.